scripts/cloud/aws/ops-ec2-snapshot-ebs-volumes.py

```diff
@@ -70,14 +72,11 @@ def main(self):
         if self.args.aws_creds_profile:
             os.environ['AWS_PROFILE'] = self.args.aws_creds_profile
 
-        regions = ebs_snapshotter.EbsSnapshotter.get_supported_regions()
-
         script_name = os.path.basename(__file__)
 
-
-        for region in regions:
-            print "Region: %s:" % region
-            ss = ebs_snapshotter.EbsSnapshotter(region.name, verbose=True)
+        if ebs_snapshotter.EbsSnapshotter.is_region_valid(self.args.region):
+            print "Region: %s:" % self.args.region
+            ss = ebs_snapshotter.EbsSnapshotter(self.args.region, verbose=True)
```

Comment:
Will this code if the region is invalid?

I don't see where it would, but I may be missing it.

I'd like the script to exit non-zero if they pass in an invalid region so that ops-runner will alert us to bad params passed in.